shared/java/org/jetbrains/jwm/MouseCursor.java

```diff
@@ -0,0 +1,18 @@
+package org.jetbrains.jwm;
+
+import org.jetbrains.annotations.*;
+
+public enum MouseCursor {
+    ARROW  (0),
+    CROSS  (1),
+    HAND   (2),
```

Comment:
macOS has three types of hands: open, closed and pointing. Let’s have more specific name here? E.g. HAND_POINTING, HAND_OPEN or HAND_CLOSED?

Reply:
Ok, but on windows they all will be mapped to IDC_HAND

Reply:
Let's leave only one, the one that Windows has?

Reply:
Windows has IDC_HAND

Reply:
Or we can do something like that in win

![image](https://user-images.githubusercontent.com/31863304/130482874-7341e02a-3757-42fb-b3a8-15c20ae868af.png)